spec.md

```diff
@@ -37,7 +37,7 @@ interface Position {
 ```js
 interface Program <: Node {
     type: "Program";
-    body: [ Statement ];
+    body: [Directive, ..., Statement ];
```

Comment:
This is not allowed by current DSL (and unclear which type `...` refers to, thus will break auto-generation of typescript and other definitions). In all the places we use smth like `[ Directive | Statement ]`).